app/app\models\user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :todo_lists
+
+  def self.set_current_user(id)
+    return if @_current_user.present?
```

Comment:
```suggestion
    return if @_current_user.present?
    
```
Adding empty line after things like guard clauses can break up code into logical paragraphs internally, subsequently more readable.

Several minor styling issues were caught with rubocop, please configure it with our organization's rubocop style to easily autofix it.